shared/local-debug.native.js

```diff
@@ -8,6 +8,8 @@ import {updateDebugConfig} from './actions/dev'
 
 const nativeBridge = NativeModules.KeybaseEngine
 
+const PERF = false
```

Comment:
Could you please add a comment to the flag explaining when to use it / what it is appropriate and not appropriate for?